docs/_releases/v0.5.15.md

```diff
@@ -0,0 +1,40 @@
+---
+layout: release
+date: 2021-05-25
+tag: v0.5.15
+---
+
+## What's New
+**General**
+- [mesheryctl] (child) View Performance Profile’s Configuration … @fallmor (#2887)
+- [mesheryctl] Validating versions and gracefully handling invalid versions @metonymic-smokey (#2980)
+- Fix `context create`: Use values from template context @navendu-pottekkat (#2998)
+- Namespace drop box to be responsive @Ayush-Khandelwal-007 (#3006)
+- Added Confirmation when Channel is Set @DARK-art108 (#3008)
+- Config file input box visibility fix @Ayush-Khandelwal-007 (#3014)
+- [Meshery UI] - Updated Meshery Provider UI @adithyaakrishna (#2927)
+- Fixed doc commands in root makefile @dhruv0000 (#2999)
+- [mesheryctl] system context list: improve msg for listing missing contexts @leecalcote (#2995)
+- Tried to rectify the issue of docs build previews trigerring on mesheryctl-changes @chinmaym07 (#2967)
+- [Docs] - Added Release Notes from v0.5.3 to v0.5.14 @adithyaakrishna (#2990)
+- [UI] Navigation Menu Icon Scale with zoom in/out  @Ayush-Khandelwal-007 (#2966)
+- Bump nokogiri from 1.11.1 to 1.11.5 in /docs @dependabot (#2987)
+
+## 🧰 Maintenance
+
+- [CI] Makefile: `make site` for conveniently building docs @leecalcote (#2993)
+- [CI] Helm: Test of helm chart releaser for remote repo @leecalcote (#2982)
+
+## 📖 Documentation
+
+- [Docs] Contributing / Project / Overview section restructure @leecalcote (#3010)
+- [Docs] Automatic Graphql Docs Generation @dhruv0000 (#2989)
+- Add contributing guidelines for mesheryctl @navendu-pottekkat (#3007)
+- [Docs] Service mesh pattern management @leecalcote (#2988)
+- [Docs]: page for `mesheryctl system update` @Blakelist7 (#2974)
+- [Docs] [WIP] Modify command reference doc pages @alphaX86 (#2985)
+
+## Contributors
```

Comment:
@Anita-ihuman Could you please remove the contributors part? i.e. Everything below Line 37 including Line 37

Reply:
IG it's a standard when a release is created. Not sure though but I've seen it many times

Reply:
Yes, in releases section of github its standard, but all the releases in docs do not have the contributors list 😶